ark-activemq/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for ark-activemq.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Please leave the `replicaCount` to 1 for the time being. Clustering is not
+# supported yet.
+replicaCount: 1
+
+image:
+  repository: 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_activemq
+  pullPolicy: IfNotPresent
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Console username and password: Please set a secure password
+adminUsername: admin
+adminPassword: admin
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
```

Comment:
@fabricetriboix i think let's keep `runAsNonRoot` as standard practice when deploying pods in the kubernetes cluster. And use `readOnlyRootFilesystem` key where required. For example for cloudconfig we require .arkcase extension as readonly.

Reply:
@naveen-armedia These as runtime options. The `ark_activemq` Docker image already ensures that ActiveMQ runs as a non-root user.

Reply:
@fabricetriboix well all the keys defined in the Values.yaml could be overridden at runtime either manually or through the script having a separate Values.yaml. This is the flexibility we have with Helm. But having those defined here would be more readable for any starters..

Reply:
@naveen-armedia OK, I will remove those values, that will make things clearer.

Reply:
Thanks @fabricetriboix